### PR TITLE
TopBarProfileMenu: update the mobile version

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/4335

# Description

Apply the latest design to the mobile version.

# Screenshots

https://user-images.githubusercontent.com/46647141/132092997-54a57a3b-3ee0-4484-9b5a-0e276cc4bf31.mp4

### CODE REVIEW

File: components/TopBarProfileMenu.js [hunk 1]
```diff
@@ -43,13 +46,89 @@ const HideGlobalScroll = createGlobalStyle`
...
+    <Box height={height}>
+      {CHANGE_LOG_UPDATES_ENABLED && (
+        <UserMenuLinkEntry as={Span} isMobileMenuLink={isMobileView} onClick={() => setShowNewsAndUpdates(true)}>
+          <FormattedMessage id="menu.newsAndUpdates" defaultMessage="News and Updates" />
+        </UserMenuLinkEntry>
+      )}
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
```
Comment (Betree) [inline components/TopBarProfileMenu.js:75]: Since the other one already triggers the query, I think it's safe to rely on the cache here

```suggestion
```

**Review by Betree — COMMENTED** [review]
Good progress! Some feedback below, mostly regarding the design, we must try to match as close as possible.

Ideally, the menu should always take the full height on mobile:
image


Some visual diffs compared to design:

image

image

File: components/TopBarProfileMenu.js [hunk 2]
```diff
@@ -43,13 +46,89 @@ const HideGlobalScroll = createGlobalStyle`
...
+  return (
+    <Box height={height}>
+      {CHANGE_LOG_UPDATES_ENABLED && (
+        <UserMenuLinkEntry as={Span} isMobileMenuLink={isMobileView} onClick={() => setShowNewsAndUpdates(true)}>
+          <FormattedMessage id="menu.newsAndUpdates" defaultMessage="News and Updates" />
+        </UserMenuLinkEntry>
+      )}
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
```
[the remark this answers is not in the document]
Comment (snitin315) [reply components/TopBarProfileMenu.js:74]: I had some conflicts, I think it's due to a bad rebase. Shouldn't be working.

**Comment by snitin315** [comment]
@Betree I have applied the latest feedback.

**Review by Betree — APPROVED** [review]
Looking good! Thanks @snitin315

**Pull request merged**